app/controllers/admin/alert_types/ratings_controller.rb

```diff
@@ -46,6 +46,20 @@ def update
         end
       end
 
+      def destroy
```

Comment:
Check for ContentManaged alert class here too, throw an error if not

Reply:
:+1: done now.  this is good for another review when you get a mo...